src/main/java/com/CezaryZal/manager/filters/validator/LoginFormValidator.java

```diff
@@ -0,0 +1,39 @@
+package com.CezaryZal.manager.filters.validator;
+
+import com.CezaryZal.exceptions.EmptyObjectException;
+import com.CezaryZal.exceptions.NullInputException;
+import com.CezaryZal.exceptions.ToShortStringException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoginFormValidator extends FormValidator {
+
+
+    void validLogin(String login) {
+        throwExceptionIfLoginIsNull(login);
+        throwExceptionIfLoginIsEmptyLogin(login);
+        isToShort(login);
+    }
+
+    private void throwExceptionIfLoginIsNull(String login) {
+        if (login == null){
+            throw new NullInputException("Input login is null");
+        }
+    }
+
+    private void throwExceptionIfLoginIsEmptyLogin(String login) {
+        if (login.isEmpty()){
+            throw new EmptyObjectException("Input login is empty");
+        }
+    }
```

Comment:
nie wiem czy jest sens rozbijać to na dwa przypadku zrobiłbym login == null || login.isEmpty i rzucił wtedy jednego exceptiona

Reply:
rozbiłem żeby szczegółowo wiedzieć co się stało.
Wiem że to raczej sprawdzane jest przez frontend ale tutaj też chciałem to sprawdzać